corehq/apps/app_manager/views/apps.py

```diff
@@ -779,8 +779,8 @@ def rearrange(request, domain, app_id, key):
             app.rearrange_forms(to_module_id, from_module_id, i, j)
         elif "modules" == key:
             app.rearrange_modules(i, j)
-    except IncompatibleFormTypeException:
-        error = _('The form is incompatible with the destination menu and was not moved.')
+    except IncompatibleFormTypeException as e:
+        error = "{}{}".format(_('The form is incompatible with the destination menu and was not moved.'), str(e))
```

Comment:
This won't have a space between the error coming from the exception and this text.

Reply:
coincidentally it works (because the exception's error message has a newline) but I can't say that it's working by design. will update.